### DISCUSSION TITLE
Is there a keyboard shortcut for setting cursor step?

### DISCUSSION THREAD

**Discussion by ecations** [schismtracker/schismtracker#439] [Q&A] [answered]
Is there a keyboard shortcut for setting cursor step (also known as edit step)?

**Comment by skope2v** [comment, marked as answer]
Yes there is, Alt 1-9. You can also set it to Alt-0 which makes it jump to the next track on the same row which is useful when writing chords.

**Comment by ecations** [comment]
Thank you very much! Maybe this shortcut could be added in the F1 manual?

**Comment by skope2v** [comment]
You're welcome. It already is in there, although perhaps not very obvious.

> Up/Down       Move up/down by the skip value (set with Alt 0-9)

**Comment by ecations** [comment]
Got it, I was trying to look for edit step and cursor step, totally missed that line!

**Comment by sagamusix** [comment]
Should we maybe turn on Discussions for this GitHub project and convert questions like this one into discussions?

**Comment by mrpapersonic** [comment]
A little late, but +1 for adding discussions, I didn't even know that was a thing.

**Comment by vcaputo** [comment]
No strong feelings either way here...

Apologies for not participating as much as I'd like to here, been busy @ new day job.
